apps/site/assets/css/_tabbed-nav.scss

```diff
@@ -0,0 +1,169 @@
+.m-tabbed-nav {
+  @include make-container();
+  @include make-container-max-widths();
+  background-color: $brand-primary;
+  display: block;
+  margin-top: $space-32;
+  padding-left: 0;
+  padding-right: 0;
+
+  @include media-breakpoint-down(sm) {
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: $space-24;
+  }
+
+  .m-tabbed-nav__items {
+    @include make-container-max-widths();
+    display: flex;
+    height: 89px;
+    margin-right: auto;
+    padding-left: $space-16;
+
+    @include media-breakpoint-down(md) {
+      height: 80px;
+      padding-left: $space-16;
+    }
+  
+    @include media-breakpoint-down(sm) {
+      height: 64px;
+      padding-left: $space-8;
+    }
+  
+    @include media-breakpoint-down(xs) {
+      height: 48px;
+      padding-left: $space-24;
```

Comment:
design nitpick: looks like the nav items should be centered rather than left aligned for the smallest breakpoint. I _think_ adding `justify-content: center` and removing the left padding should accomplish that here.